FireBrick/UI/AddTechnologyViewController/AddTechnologyViewController.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  AddTechnologyViewController.swift
+//  FireBrick
+//
+//  Created by metoSimka on 27/05/2019.
+//  Copyright © 2019 metoSimka. All rights reserved.
+//
+
+import UIKit
+import SwiftEntryKit
+import Firebase
+import FirebaseFirestore
+
+class AddTechnologyViewController: UIViewController {
+    
+    var docRef: DocumentReference!
+    var db: Firestore?
+    
+    @IBOutlet weak var addButton: UIButton!
+    @IBOutlet weak var nameTextField: UITextField!
+    @IBOutlet weak var docTextField: UITextField!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        db = Firestore.firestore()
+        nameTextField.delegate = self
+        docTextField.delegate = self
+        updataButtonState()
+    }
+    
+    @IBAction func cancel(_ sender: UIButton) {
+        SwiftEntryKit.dismiss()
+    }
+    
+    @IBAction func add(_ sender: UIButton) {
+        writeData()
+    }
+    
+    func writeData() {
+        guard let name = nameTextField.text else {
+            return
+        }
+        guard let doc = docTextField.text else {
+            return
+        }
+        guard name.count > 0 && doc.count > 0 else {
+            return
+        }
+        
+        let dataToSave: [String: Any] = ["Name": name, "Documentation": doc]
+        db!.collection("Technology").addDocument(data: dataToSave, completion: { (error) in
+            guard error == nil else {
+                print("there must be ALERT!")
+                return
+            }
+                SwiftEntryKit.dismiss()
+        })
+    }
+    
+    func isFildsAreEmpty() -> Bool {
```

Comment:
need to make proper field validation
func formIsValid() -> Bool
func nameIsValid() -> Bool
func documentationIsValid() -> Bool